public/app/features/dashboard/services/TimeSrv.ts

```diff
@@ -24,7 +24,7 @@ export class TimeSrv {
   time: any;
   refreshTimer: any;
   refresh: any;
-  previousAutoRefresh: any;
+  refreshActive = true;
```

Comment:
Nit: `refreshActive` wasn't super clear to me when I first went through the change. Maybe an alternative name could be 
- `canAutoRefresh`
- `disableAutoRefresh`